lib/flagutil/secure_url.go

```diff
@@ -0,0 +1,106 @@
+package flagutil
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"sync/atomic"
+	"unicode"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/fasttime"
+)
+
+// NewSecureURL returns new `url` flag with the given name and description.
+//
+// The url value is hidden when calling SecureURL.String() for security reasons,
+// since the returned value can be put in logs.
+// Call SecureURL.Get() for obtaining the real url value.
+func NewSecureURL(name, description string) *SecureURL {
```

Comment:
Maybe we should go with just `URL` without `Secure` prefix. It is unlikely we'll have `URL` and `SecureURL` types.

Reply:
Agree. I have addressed this in the latest commit.